lib/ui/views/navigation/navigation_viewmodel.dart

```diff
@@ -32,7 +33,20 @@ class NavigationViewModel extends IndexTrackingViewModel {
 
     if (prefs.getInt('themeMode') == null) {
       await prefs.setInt('themeMode', 0);
+      await DynamicTheme.of(context)!.setTheme(0);
     }
+
+    // Force disable Material You on Android 11 and below
+    if (DynamicTheme.of(context)!.themeId.isOdd) {
+      const int ANDROID_12_SDK_VERSION = 31;
+      final AndroidDeviceInfo info = await DeviceInfoPlugin().androidInfo;
+      if (info.version.sdkInt < ANDROID_12_SDK_VERSION) {
```

Comment:
I would personally remove the variable `ANDROID_12_SDK_VERSION` because it's being used once

```suggestion
      final AndroidDeviceInfo info = await DeviceInfoPlugin().androidInfo;
      if (info.version.sdkInt < 31) {
```

Reply:
It is to make the code more clear as 12 makes more sense than 31